Database: replace level 'content' with Tile entity

mrocha51248 opened the pull request.

rocambille commented on database.sql:
```diff
@@ -24,9 +24,11 @@ DROP TABLE IF EXISTS `level`;
+  `name` varchar(30) NOT NULL,
   `description` text,
-  `content` varchar(2000) DEFAULT NULL,
+  `content` longtext NOT NULL,
+  `width` int NOT NULL,
+  `height` int NOT NULL,
```
if you're working on the database, it's time to get rid of the "01" and use a cell entity

mrocha51248 replied:
done, though it'll be more annoying to review with everything done at once ;)

rocambille commented on src/Controller/GameController.php:
```diff
@@ -44,13 +45,15 @@ public function index(?string $action): string
+        $tileManager = new TileManager();
+        $tiles = $tileManager->selectAllByLevelId($levelId);
+
+        $this->move($tiles, $action ?? "");
```
you may just compute the potential new position, then search for a tile of the level at the potential position which is not a wall: if it exists, accept the move; otherwise keep the old position

rocambille commented on src/Controller/GameController.php:
```diff
@@ -44,13 +45,15 @@ public function index(?string $action): string
 
-        $this->move($cells, $action ?? "");
+        $tileManager = new TileManager();
+        $tiles = $tileManager->selectAllByLevelId($levelId);
```
you may just ask for the tile of the level that are in the wanted radius ;)

mrocha51248 commented:
addressed comments. more work will be required to get rid of ``generateViewpoint`` and ``generateCellDetails`` (actually implement ``start`` and ``finish`` tiles and update views). that will be done in follow up PRs
